.php_cs.dist

```diff
@@ -0,0 +1,60 @@
+<?php
+
+$header = <<<'HEADER'
+This file is part of the Sonata Project package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+HEADER;
+
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
+    ->exclude('Tests/Fixtures')
+    ->exclude('tests/Fixtures')
+    ->exclude('Resources/skeleton')
+    ->exclude('Resources/public/vendor')
+;
+
+return PhpCsFixer\Config::create()
+    ->setFinder($finder)
+    ->setRiskyAllowed(true)
```

Comment:
I'm not sure this should be true... will it create BC-breaks by e.g. changing signatures to add `void`, things like that?

Reply:
I set the `setRiskyAllowed` option to `true` to use the options below :
- declare_strict_types
- modernize_types_casting
- psr4
- strict_comparison
- strict_param

But is this really necessary for the different Sonata project?

Reply:
The `setRiskyAllowed` can stay here, but we have to filter the risky fixers.

For example, `strict_param` and `strict_comparison` might introduce BC break on current stables.